spec/keyboard-commands.spec.js

```diff
@@ -110,5 +110,30 @@ describe('KeyboardCommands TestCase', function () {
             });
             expect(editor.execAction).not.toHaveBeenCalled();
         });
+
+        it('should not execute when the command key are false', function () {
```

Comment:
Can you add another test case (or another assertion in this test case) where if `event.shiftKey` is `true`, it should NOT block the code from executing?